### PR TITLE
Add an example for definition file structure of 'New-AzSynapseManagedPrivateEndpoint'

### PR DESCRIPTION
**Pull request by yanjungao718**
## Description

Add an example for definition file structure of 'New-AzSynapseManagedPrivateEndpoint'

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Synapse/Synapse/help/New-AzSynapseManagedPrivateEndpoint.md [hunk 1]
```diff
@@ -79,7 +79,16 @@ Accept wildcard characters: False
...
+The JSON file path. Below is the json file structure example:
+```json
+{
+	"name": "ContosoManagedPrivateEndpoint",
+	"properties": {
+		"privateLinkResourceId": "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/contosoResourceGroup/providers/Microsoft.Storage/storageAccounts/contosoStorageAccount",
+		"groupId": "file"
+	}
+}
+```
```
Comment (wonner) [inline src/Synapse/Synapse/help/New-AzSynapseManagedPrivateEndpoint.md:91]: Please update the HelpMessage of DefinitionFile for this command. If not, I suppose these help messages will be lost if you regenerate this help doc.

Comment (BethanyZhou) [reply src/Synapse/Synapse/help/New-AzSynapseManagedPrivateEndpoint.md:91]: Another way to add sample file in example is adding it as comments in script, see [Set-AzKeyVaultKeyRotationPolicy]( https://github.com/Azure/azure-powershell/blob/main/src/KeyVault/KeyVault/help/Set-AzKeyVaultKeyRotationPolicy.md#example-1-sets-key-rotation-policy-by-json-file)

Comment (yanjungao718) [reply src/Synapse/Synapse/help/New-AzSynapseManagedPrivateEndpoint.md]: added the sample json to example1 as a comment.

File: src/Synapse/Synapse/ChangeLog.md [hunk 2]
```diff
@@ -19,6 +19,7 @@
 -->
 
 ## Upcoming Release
+* Added example for definition file structure of 'New-AzSynapseManagedPrivateEndpoint'
```
Comment (BethanyZhou) [inline src/Synapse/Synapse/ChangeLog.md:22]: No need for change log if only docs changes.

Comment (yanjungao718) [reply src/Synapse/Synapse/ChangeLog.md]: ok, thanks, reverted.

**Review by BethanyZhou — APPROVED** [review]
Looks good to me

**Pull request merged**